langchain/src/retrievers/parent_document.ts

```diff
@@ -54,10 +61,18 @@ export class ParentDocumentRetriever extends MultiVectorRetriever {
     this.idKey = fields.idKey ?? this.idKey;
     this.childK = fields.childK;
     this.parentK = fields.parentK;
+    this.retriever = fields.retriever;
   }
 
   async _getRelevantDocuments(query: string): Promise<Document[]> {
-    const subDocs = await this.vectorstore.similaritySearch(query, this.childK);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let subDocs: Document<Record<string, any>>[] = [];
+    if (this.retriever) {
+      subDocs = await this.retriever.getRelevantDocuments(query);
```

Comment:
Pass `childK`?

Reply:
No param for that in `getRelevantDocuments` instead you'd pass it when you initialize the class outside of `ParentDocumentRetriever`

ex:
```typescript
const scoreThresholdRetriever = ScoreThresholdRetriever.fromVectorStore(
  vectorstore,
  {
    ...,
    maxK: 1,
  }
);
```

Reply:
Oh, right thanks